comfy_cli/cmdline.py

```diff
@@ -133,14 +133,31 @@ def install(
     workspace_manager.set_recent_workspace(workspace_path)
 
 
-def update(self):
+@app.command(help="Stop background ComfyUI")
+@tracking.track_command()
+def update(
+    ctx: typer.Context, target: str = typer.Argument("comfy", help="[all|comfy]")
+):
+    if target not in ["all", "comfy"]:
+        typer.echo(
+            f"Invalid target: {target}. Allowed targets are 'all', 'comfy'.",
+            err=True,
+        )
+        raise typer.Exit(code=1)
+
     _env_checker = EnvChecker()
-    print(f"Updating ComfyUI in {self.workspace}...")
-    os.chdir(self.workspace)
-    subprocess.run(["git", "pull"], check=True)
-    subprocess.run(
-        [sys.executable, "-m", "pip", "install", "-r", "requirements.txt"], check=True
-    )
+    comfy_path = workspace_manager.get_workspace_comfy_path(ctx)
```

Comment:
Once I merge https://github.com/Comfy-Org/comfy-cli/pull/19, you can use workflow_manager.workspace_path